vm/gvisor/gvisor.go

```diff
@@ -94,7 +100,11 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 		caps += "\"" + c + "\""
 	}
 	name := fmt.Sprintf("%v-%v", pool.env.Name, index)
-	vmConfig := fmt.Sprintf(configTempl, imageDir, caps, name)
+	memoryLimit := pool.cfg.MemoryTotalBytes / uint64(pool.Count())
+	if pool.cfg.MemoryTotalBytes == 0 {
+		memoryLimit = osutil.SystemMemorySize()
```

Comment:
osutil.SystemMemorySize() / uint64(pool.Count()) ?

Reply:
This would impose a limit on instances where we don't want to impose a limit. (I don't know if there are cases like this but empirically we only have issues with ARM instances, so I say let's only impose a limit where it's necessary).

Reply:
Is there any magic words like max, unlimited, etc?

Reply:
Yeah -1 indicates unlimited